apps/api/src/app/workflows-v1-deprecated/notification-template.controller.ts

```diff
@@ -191,7 +195,8 @@ export class NotificationTemplateController {
         data: body.data,
         __source: query?.__source,
         type: WorkflowTypeEnum.REGULAR,
-      })
+        origin: WorkflowOriginEnum.NOVU,
+      } as CreateWorkflowCommand)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** why do we need to coerce the type here? This could obfuscate type errors‏

Reply:
it's not coercing, it's enforcing, the .create() accepts some kind of any and the type check doesn't work well, need to be fixed across the board